examples/manual_routes/src/main.rs

```diff
@@ -52,10 +56,15 @@ fn get_upload(_: &Request, _: Data) -> Outcome<'static> {
 }
 
 fn not_found_handler<'r>(_: Error, req: &'r Request) -> response::Result<'r> {
-    format!("Couldn't find: {}", req.uri()).respond()
+    let message = format!("Couldn't find: {}", req.uri());
```

Comment:
Sorry I didn't get to your message! We should do:

```rust
status::Custom(Status::NotFound, format!("Couldn't find: {}", req.uri())).respond()
```

You'll need to bring back the `Responder` `use` for this to work.